pulp_rpm/app/models/distribution.py

```diff
@@ -253,7 +256,7 @@ class Addon(BaseModel):
         DistributionTree, on_delete=models.CASCADE, related_name='addons'
     )
     repository = models.ForeignKey(
-        Repository, on_delete=models.PROTECT, related_name='addons'
+        Repository, on_delete=models.DO_NOTHING, related_name='addons'
```

Comment:
Silly question, but why not delete the `Addon` when the repository it refers to no longer exists?

Reply:
I think PROTECT doesn't allow to delete a repo if it belongs to any addon.
https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.PROTECT

Reply:
To confirm, the on_delete behavior will be triggered when the repository is deleted, yes?
In that case should the `Addon` referring to it, go with it? If so that would be `CASCADE`?

I'm not sure what is right, so these questions could be off-base.